src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpPreferStreamAsyncMemoryOverloads.Fixer.cs

```diff
@@ -54,18 +68,70 @@ protected override bool IsSystemNamespaceImported(IReadOnlyList<SyntaxNode> impo
 
         protected override bool IsPassingZeroAndBufferLength(SemanticModel model, SyntaxNode bufferValueNode, SyntaxNode offsetValueNode, SyntaxNode countValueNode)
         {
-            return
```

Comment:
I decided to split this method into multiple `if` clauses to make it easier to debug.